tests/e2e/vmgroups_test.go

```diff
@@ -46,6 +47,7 @@ type VmGroupTest struct {
 	volName1      string
 	volName2      string
 	volName3      string
+	volName4      string
```

Comment:
This isn't because of your change but this should have been a slice instead. You might wanna handle it if it is trivial.